src/modules/fancyzones/editor/FancyZonesEditor/App.xaml.cs

```diff
@@ -21,6 +21,32 @@ namespace FancyZonesEditor
     /// </summary>
     public partial class App : Application
     {
+        // Localizable strings
+        private const string CrashReportMessageBoxTextPart1 = "Error logged to ";
```

Comment:
End user readable tags should be in the resx.

- CrashReportMessageBoxTextPart1 
- CrashReportMessageBoxTextPart2 
- CrashReportMessageBoxCaption

Reply:
Moved to resource file.